deployments/examples/ocis_wopi/.env

```diff
@@ -45,7 +45,7 @@ OCIS=:ocis.yml
 OCIS_DOCKER_IMAGE=
 
 # The oCIS container version.
-# Defaults to "latest". This will point to the latest stable tag.
+# Defaults to "latest". Use the latest stable tag like 5.0.5.
```

Comment:
```suggestion
# Defaults to "latest". This will point to the latest production tag.
```
Originally, this was: `This will point to the latest stable tag.`. After discussing with @micbar I turned it back but changed the name from `stable` to `production` as it follows the intro on docker hub.